cmds/core/strace/strace.go

```diff
@@ -25,15 +25,17 @@ import (
 	"github.com/u-root/u-root/pkg/strace"
 )
 
-var (
```

Comment:
Any reason for cmdUsage to be a var rather than const?

Reply:
nope not really